servant/stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: nightly-2020-05-06
+resolver: lts-12.26
```

Comment:
I imagine that for `servant` you could actually use a more recent resolver

Reply:
Oh sorry. I simply used the same one `spock` was using. Feel free to change it to a  more suitable one.

Reply:
I pushed on master with the most recent resolver. Next week I'll try to upgrade also the spock folder (it uses 12.26 because it's no more on Stackage)